docs/07-advanced-usage.md

```diff
@@ -19,6 +19,24 @@ Browsers only support importing by URL, so importing a package by name (ex: `im
   ]
 ```
 
+**If you use TypeScript with Snowpack, you can use a [ttypescript](https://github.com/cevek/ttypescript) plugin called [@magic-works/ttypescript-browser-like-import-transformer](https://github.com/Jack-Works/ttypescript-browser-like-import-transformer) to support package name imports.** The transformer reads any import and export declarations in your source code and rewrites them to full `"/web_modules/${PACKAGE_NAME}.js"` URLs that run in the browser. This way, you can keep using the package name imports that you're used to without needing a full web app bundler. Check out our guide below.
```

Comment:
you can remove "Check out our guide below." and get rid of some of the copy paste from the section above.

I'm about to do a pass of the docs anyway to edit the config docs that just went in, so I'll just merge this and then fix these up as a part of this.